docs/examples/example-grid-tariff-capacity-part.md

```diff
@@ -575,9 +581,11 @@ const MAX_COUNTING = 3; // Number of days to calculate month
 const BUFFER = 0.5; // Closer to limit increases level
 const SAFE_ZONE = 2; // Further from limit reduces level
 const ALARM = 8; // Min level that causes status to be alarm
+const MIN_TIMELEFT = 3*60; //Min level for time left
 
```

Comment:
Just wondering if 3 minutes is a little long. What if there is a large consumption at the last minute, then the required reduction will be less than what is actually necessary, right? Or am I thinking wrong here?
How much time is required to get rid of that spike problem?

Reply:
I can do some calculations/simulations of the scenario you mention. But it’s a variable that can be changed. By using 3 minutes, my increase possible is maxing out at around 160-170 kW.. Another approach could be to limit the values based on the maximum power intake.

Reply:
I did some simulations - if the hourEstimate is above the current step the reductionRecommended will dramatically increase as time is reaching zero without the time limit. How can we reduce more than maximum of the power intake? By using 3 minutes for the MIN_TIMELEFT the reductionRecommended were flattened around 50kW for the last 3 minutes.

However - for the reductionRequired, the values wasn't increasing as much as I expected with low values for MIN_TIMELEFT. Even for 10 seconds the value is below 170 kW.